packages/testing/addon/mirage-graphql/schema.graphql

```diff
@@ -759,7 +1048,6 @@ type CompleteTaskFormTask implements Task & Node {
   slug: String!
   name: String!
   description: String
-  type: TaskType!
```

Comment:
Why is TaskType no longer needed?

Reply:
This information is in the `__typename`. Not sure why the backend removed this explicitly.